cmd/kops/rollingupdatecluster.go

```diff
@@ -189,7 +189,7 @@ func NewCmdRollingUpdateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().DurationVar(&options.PostDrainDelay, "post-drain-delay", options.PostDrainDelay, "Time to wait after draining each node")
 	cmd.Flags().BoolVarP(&options.Interactive, "interactive", "i", options.Interactive, "Prompt to continue after each instance is updated")
 	cmd.Flags().StringSliceVar(&options.InstanceGroups, "instance-group", options.InstanceGroups, "List of instance groups to update (defaults to all if not specified)")
-	cmd.Flags().StringSliceVar(&options.InstanceGroupRoles, "instance-group-roles", options.InstanceGroupRoles, "If specified, only instance groups of the specified role will be updated (e.g. Master,Node,Bastion)")
+	cmd.Flags().StringSliceVar(&options.InstanceGroupRoles, "instance-group-roles", options.InstanceGroupRoles, "If specified, only instance groups of the specified role will be updated (e.g. Master,Node,Bastion,Apiserver)")
```

Comment:
Consider capitalizing the new one differently:
```suggestion
	cmd.Flags().StringSliceVar(&options.InstanceGroupRoles, "instance-group-roles", options.InstanceGroupRoles, "If specified, only instance groups of the specified role will be updated (e.g. Master,Node,Bastion,APIServer)")
```
I suppose this is predicated on whether we think that the term is "apiserver" (as occurs sometimes in Kubernetes code, but not often in documentation) or "API Server." I favor the latter.

Reply:
Are these roles ordered from most to least likely to be pertinent here? I see that the order differs here, in _kops_create_instancegroup.md_, and in _kops_rolling-update_cluster.md_.